src/controller/eme-controller.js

```diff
@@ -14,78 +14,79 @@ const { XMLHttpRequest } = window;
 
 const MAX_LICENSE_REQUEST_FAILURES = 3;
 
-/**
- * @see https://developer.mozilla.org/en-US/docs/Web/API/MediaKeySystemConfiguration
- * @param {Array<string>} audioCodecs List of required audio codecs to support
- * @param {Array<string>} videoCodecs List of required video codecs to support
- * @param {object} drmSystemOptions Optional parameters/requirements for the key-system
- * @returns {Array<MediaSystemConfiguration>} An array of supported configurations
- */
-
-const createWidevineMediaKeySystemConfigurations = function (audioCodecs, videoCodecs, drmSystemOptions = {}) { /* jshint ignore:line */
+const buildKeySystemConfigurations = function (audioCodecs, videoCodecs, drmSystemOptions = {}) {
   const baseConfig = {
     // initDataTypes: ['keyids', 'mp4'],
     // label: "",
     // persistentState: "not-allowed", // or "required" ?
-    // distinctiveIdentifier: "not-allowed", // or "required" ?
+    // distinctiveIdentifier: "not-allowed", // "not-allowed" or "required" ?
     // sessionTypes: ['temporary'],
-    audioCapabilities: [
-      // { contentType: 'audio/mp4; codecs="mp4a.40.2"' }
-    ],
-    videoCapabilities: [
-      // { contentType: 'video/mp4; codecs="avc1.42E01E"' }
-    ]
+    // audioCapabilities: [
+    // { contentType: 'audio/mp4; codecs="mp4a.40.2"' }
+    // ],
+    // videoCapabilities: [
+    // { contentType: 'video/mp4; codecs="avc1.42E01E"' }
+    // ]
   };
 
+  if (drmSystemOptions.distinctiveIdentifier) {
```

Comment:
or, we can extract `robustness` property and put the rest properties on `baseConfig`:

```js
const { audioRobustness, videoRobustness, ...others } = drmSystemOptions;
baseConfig = { ...baseConfig, ...others };
...
```

Reply:
For PS5 we cannot have robustness setting set at all if it is not being used. So the way I have is fine IMO.